Bottom: cc9756bbdca9639c9e616385a447d2628b32043d
Top:    92d4197ffd77dee8899a0c4421f5af17ac683d31
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 17:30:12 +0000

Refresh of play-with-quad-edge-data

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 3e94720..effb634 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1,6 +1,5 @@
 #include <stdio.h>
 #include <stdlib.h>
-#include <assert.h>
 #include <stdbool.h>
 
 #include <glib.h>
